docs/faq.md

```diff
@@ -0,0 +1,63 @@
+# FAQ
+
+## What´s the difference between `>` and `$`?
+
+`>` or `connect-terminal` runs its argument
+(which by default is your shell)
+inside a terminal,
+while `$` or `connect-shell` simply runs its argument.
+
+Some programs must be run inside a terminal,
+while others spawn their own graphical window.
+Use `>` for the first type and `$` for the second.
+
+## What´s the purpose of `&` or `connect-detach`?
+
+`connect-detach` detaches (closes) the current client
+(leaving you in a terminal if you launched Kakoune there) and
+creates a file named `connect.sh` which you can run with `sh connect.sh`
+to start a shell connected to that client´s server.
+The file is automatically erased when it´s sourced.
+
+**Note 1**: The Kakoune server is deleted if you detach the last client and
+you didn´t start it in daemon mode.
+
+**Note 2**: The implementation uses Kakoune´s `quit!` command instead of `quit`.
```

Comment:
Apostrophe is `’`.

Reply:
I mistaked it for `´` (acute accent). The fact that is is called [`Right Single Quotation Mark` ][https://codepoints.net/U+2019] also confused me. It should be fixed now.

Reply:
It looks present everywhere.

Reply:
> It looks present everywhere.

Srry, forgot to add before ammending. Reupdated.